test/Elastic.Apm.Tests/SerializationTests.cs

```diff
@@ -115,6 +125,22 @@ public void LabelsTruncationSpanContext()
 			Assert.Equal("...", deserializedContext[intakeApiLabelsPropertyName].Value<JObject>()["foo"].Value<string>().Substring(1021, 3));
 		}
 
+		/// <summary>
+		/// Makes sure that labels with `null` don't cause exception during serialization
+		/// </summary>
+		[Fact]
+		public void LabelWithNullValueShouldBeCaptured()
+		{
+			var context = new SpanContext();
+			context.Labels["foo"] = null;
+
+			var json = SerializePayloadItem(context);
+			var deserializedContext = JsonConvert.DeserializeObject(json) as JObject;
+
+			Assert.NotNull(deserializedContext);
+			deserializedContext["foo"].Should().BeNull();
```

Comment:
This test passes even though `Elastic.Apm.Report.Serialization.LabelsJsonConverter.WriteJson` is not correct because the assert above is not correct neither, The correct assert is `deserializedContext["tags"]["foo"].Value<string>().Should().BeNull();`

Reply:
Fixed